WordPressLoginFlow/build.gradle

```diff
@@ -38,11 +38,11 @@ repositories {
 }
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
 
     defaultConfig {
-        minSdkVersion 21
-        targetSdkVersion 29
+        minSdkVersion 24
```

Comment:
⚠️ Just a headsup that WooAndroid still uses api 21 -> so if we want to bump it, we might want to consider bumping it only to 21 so detekt/lint/AS keeps notifying us about issues. Wdyt?

Reply:
Thank you @malinajirka for the heads-up.  I've now reverted back to 21